CMakeLists.txt

```diff
@@ -52,4 +52,17 @@ if(WIN32)
 endif()
 
 #METABLE_DEFINE_TEST(metable_server SRC_FILES src/server/server_main.cc LINKS gflags)
-METABLE_DEFINE_TEST(metable_server SRC_FILES src/server/server_main.cc)
+#METABLE_DEFINE_TEST(metable_server SRC_FILES src/server/server_main.cc LINKS common_lib)
```

Comment:
Why comment out this line? I believe we should produce the metrable_server binary.

Reply:
yes,you are right.I was wrong to think about it.

Reply:
You're right. I only noticed the modification of gtest. I didn't look back at this.